spec/linter-stylelint-spec.js

```diff
@@ -301,4 +303,33 @@ describe('The stylelint provider for Linter', () => {
       );
     });
   });
+
+  describe('works with JS files (language-babel) with Styled Components and', () => {
+    beforeEach(() => {
+      atom.config.set('linter-stylelint.disableWhenNoConfig', false);
+      waitsForPromise(() => atom.packages.activatePackage('language-babel'));
```

Comment:
Does this _need_ `language-babel`? I'd like to avoid bringing that in here if possible. If it is _required_, then it needs to be added to the list of pre-installed packages in the CI scripts.